src/components/views/rooms/EventTile.js

```diff
@@ -282,18 +284,40 @@ export default createReactClass({
         }
     },
 
+    onUserVerificationChanged: function(userId, _trustStatus) {
+        if (userId === this.props.mxEvent.getSender()) {
+            this._verifyEvent(this.props.mxEvent);
+        }
+    },
+
     _verifyEvent: async function(mxEvent) {
         if (!mxEvent.isEncrypted()) {
             return;
         }
 
+        // If we directly trust the device, short-circuit here
         const verified = await this.context.isEventSenderVerified(mxEvent);
+        if (verified) {
+            this.setState({
+                verified: "verified",
+            }, () => {
+                // Decryption may have caused a change in size
+                this.props.onHeightChanged();
+            });
+            return;
+        }
+
+        const eventSenderTrust = await this.context.checkEventSenderTrust(mxEvent);
+        if (!eventSenderTrust) {
+            this.setState({
+                verified: "unknown",
+            }, this.props.onHeightChanged); // Decryption may have cause a change in size
```

Comment:
this isn't decryption though?

Reply:
here I'm just copying from elsewhere in the function where I assume it was there for good reason. i think it's that if the element is decrypted then this causes different elements to render. I can take it out, if you like, but it migth be a regression.